src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={
+    classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )
+  }
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user && <UserInfo user={todo.user} />}
```

Comment:
```suggestion
    {todo.user && (
      <UserInfo user={todo.user} />
    )}
```
it's better to write this statements on separate lines for better readability